index.html

```diff
@@ -0,0 +1,1729 @@
+            </div>
+
+            <div class="debate-item3 d-flex flex-column pt-3 border-top mb-2">
+              <div class="flex d-flex">
+                <div class="headsot pr-3">
+                  <img src="resources/chuck-devore.png" alt="" />
+                </div>
+                <div class="info">
+                  <div class="title">
+                    <a href="#">We Cannot Wait to Confront Communist China </a>
+                  </div>
+                  <div class="byline-option">BY CHUCK DEVORE</div>
+                </div>
+              </div>
+            </div>
+
+            <div class="debate-item3 d-flex flex-column pt-3 border-top mb-1">
+              <div class="flex d-flex">
+                <div class="headsot pr-3">
+                  <img src="resources/bonnie-kristian.png" alt="" />
+                </div>
+                <div class="info">
+                  <div class="title">
+                    <a href="#"
+                      >Trump's China Hawks Should Prioritize Helping America,
+                      Not Hurting Beijing
+                    </a>
+                  </div>
+                  <div class="byline-option">BY BONNIE KRISTIAN</div>
+                </div>
+              </div>
+            </div>
+
+            <div class="debate-item3 d-flex flex-column pt-3 border-top mb-2">
+              <div class="flex d-flex">
+                <div class="headsot pr-3">
+                  <img
+                    src="resources/michael-makovsky-charlie-perkins.png"
+                    alt=""
+                  />
+                </div>
+                <div class="info">
+                  <div class="title">
+                    <a href="#">Why Israel Is the Ideal Ally for This Moment</a>
+                  </div>
+                  <div class="byline-option">
+                    BY MICHAEL MAKOVSKY AND CHARLES PERKINS
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+
+          <div class="newsletter container-fluid mt-5 row">
+            <div class="newsletter-text d-flex align-items-center">
+              <span class="icon mb-2"></span>
+              <h3 class="text-news">
+                Start your day with our top 5 articles
+              </h3>
+            </div>
+            <form class="signup-form">
+              <input
+                type="text"
+                class="text-input"
+                placeholder="Email address"
+              />
+              <input
+                type="submit"
+                class="submit-form mt-2"
+                value="free sign up"
+              />
+            </form>
+          </div>
+        </div>
+      </div>
+
+      <div class="magazine container-fluid">
+        <div
+          class="magazine-title mb-2 p-3 border-bottom d-flex align-items-center justify-content-center"
+        >
+          <span class="icon"></span>
+          IN THE MAGAZINE
```

Comment:
- [ ] There should be four columns for medium devices in this section.

Source | Clone
-- | --
![image](https://user-images.githubusercontent.com/57414789/83581073-61a66080-a503-11ea-9af7-adddf5d7c56b.png) | ![image](https://user-images.githubusercontent.com/57414789/83581052-58b58f00-a503-11ea-9d0b-f6c1f3312fce.png)